homeassistant/components/unifi/device_tracker.py

```diff
@@ -80,6 +81,23 @@
     controller = hass.data[unifi.DOMAIN][controller_id]
     tracked = {}
 
+    registry = await entity_registry.async_get_registry(hass)
+
+    # Restore clients that is not a part of active clients list.
+    for entity in registry.entities.values():
+
+        if entity.config_entry_id == config_entry.entry_id and \
+                entity.domain == DOMAIN:
+
+            mac, _ = entity.unique_id.split('-', 1)
```

Comment:
Maybe we should use the device registry instead, since we store the mac address there as part of the connection? That seems more natural to me as we don't need to split a string. Is the mac address format in device registry intact vs `client.mac`?

Reply:
You don't get the relation with device tracker domain with device registry, and the client.mac isn't unique to device tracker entities.